schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -221,7 +221,6 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-07-01/Microsoft.Media.json#/resourceDefinitions/mediaServices_streamingPolicies" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-07-01/Microsoft.Media.json#/resourceDefinitions/mediaServices_transforms" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-07-01/Microsoft.Media.json#/resourceDefinitions/mediaServices_transforms_jobs" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2016-09-01/Microsoft.Authorization.json#/resourceDefinitions/locks" },
```

Comment:
The error should be caused by deleting this line. With the `2016-09-01`, schema validator find one possible valid value is `Microsoft.Network/publicIPAddresses`, and then use it to validate.

Reply:
I reverted that change and it seemed to fix the issue.
All tests are passing again

Reply:
@leni-msft forgot to @

Reply:
ok, LGTM